KosherCocoa/Library/Core/Solar/KCAstronomicalCalendar.m

```diff
@@ -173,7 +174,7 @@ - (double)UTCSunrise:(double)zenith
 
 - (double)UTCSeaLevelSunrise:(double)zenith
 {
-    double sunrise = [((KCSunriseAndSunsetCalculator *)self.astronomicalCalculator) UTCSunriseForDate:self.workingDate andZenith:zenith adjustForElevation:NO]; 
+    double sunrise = [((KCNOAACalculator *)self.astronomicalCalculator) UTCSunriseForDate:self.workingDate andZenith:zenith adjustForElevation:NO]; 
```

Comment:
I would relook at this method because the UTCSeaLevelSunset method does not need the type for the calendar. Maybe we could avoid using the type if we copied that method.

Reply:
We can use the superclass, `KCAstronomicalCalculator` here.